build/org.eclipse.cdt.managedbuilder.core/src/org/eclipse/cdt/managedbuilder/internal/core/CommonBuilder.java

```diff
@@ -509,11 +509,7 @@ private IProject[] build(int kind, IProject project, IBuilder[] builders, boolea
 				}
 			}
 
-			for (int i = 0; i < num; i++) {
-				if (isGenerateFileOptionEnabled()) {
-					CompilationDatabaseGenerator generator = new CompilationDatabaseGenerator(getProject(), activeCfg);
-					generator.generate();
-				}
+			for (int i = 0; i < num; i++) {		
```

Comment:
The code cleanliness checks are going to fail because of trailing whitespace here
```suggestion
			for (int i = 0; i < num; i++) {
```